WTWSMS/common/tributary_types/00_tributary_types.txt

```diff
@@ -0,0 +1,39 @@
+#For tributary types with breaks_at_suzerain_death = yes add the type to JD.60015 for the notification event
+
+foederati = {
+	tributary_name = TRIBUTARY_DEFAULT_NAME
+	tributary_plural = TRIBUTARY_DEFAULT_PLURAL
+	suzerain_name = SUZERAIN_DEFAULT_NAME
+	tribute_name = TRIBUTARY_DEFAULT_TRIBUTE
+	breaks_at_suzerain_death = yes
+	breaks_at_tributary_death = no
+	suzerain_must_defend = no
+	tributary_can_be_summoned = yes
+	tributary_must_defend = yes
+	tributaries_can_infight = yes
+	suzerain_can_join_infighting = yes
+	tributaries_can_unite = no
+	breaks_on_realm_change = yes
+	icon = 7
+	cbs_against_suzerain = {
+		free_tributary_cb
+	}
+	income_tribute_percentage = {
+		value = 0.4
+	}
+	reinforce_tribute_percentage = {
+		value = 0
+	}
+	prestige_to_suzerain = {
+		value = 0
+	}
+	prestige_to_tributary = {
+		value = 0
+	}
+	piety_to_suzerain = {
+		value = 0
+	}
+	piety_to_tributary = {
+		value = 0
+	}
+}
```

Comment:
This file shouldn't be separate from `WtWSMS_tributary_types.txt`, otherwise you risk to overwrite all the vanilla tributaries, which we don't want as they are used on the map.

Reply:
moved to WtWSMS_tributary_types.txt

Reply:
Good!